knowage-core/src/main/java/it/eng/spagobi/api/common/AbstractDataSetResource.java

```diff
@@ -1121,7 +1121,7 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
 		}
 	}
 
-	public Response execute(String label, String body) {
+	protected Response execute(String label, String body) {
```

Comment:
Do you think it's possible to use private? If there is no inheritance I would prefer the private modifier.

Reply:
This one can't be private, it is being used in DataSetResource.java (two files, one of them is v2).